src/Uno.UI.RemoteControl.Host/Extensibility/AddIns.cs

```diff
@@ -17,29 +17,54 @@ public class AddIns
 	public static IImmutableList<string> Discover(string solutionFile)
```

Comment:
can we add unit tests for these changes? for scenarios like empty frameworks, invalid solution files, command execution failures, for example

Reply:
We don't have testing yet for dev-server. Will need it at some point, but not part of this PR